obstacledetection/src/main/java/com/caneseeproject/obstacledetection/ObstacleDetectionInterface.kt

```diff
@@ -41,7 +41,7 @@ interface ObstacleDetector : Sensor<ODReading, ODControl>{
     /**
      * Provides flow of high level sensor readings
      */
-    fun getCaneData(): Flow<ODReading>
+    fun speak(): Flow<ODReading>
```

Comment:
This name is very ambiguous; how can this module `speak`? It's okay for TTs, but in this module, this name is very confusing to your API users.

Reply:
I really thought the same , but i couldn't think of another one for this function . I'll change it.

Reply:
You can remove it entirely if it does the same thing as `readings`.